profiles/kentik_snmp/cisco/cisco-asr.yml

```diff
@@ -93,32 +93,3 @@ sysobjectid:
   - 1.3.6.1.4.1.9.1.2679  # Cisco 92012SZA ASR
   - 1.3.6.1.4.1.9.1.2705  # Cisco 92020SZM ASR
   - 1.3.6.1.4.1.9.6.1.23.3.13.0.4  # Cisco RV340 Dual WAN Gigabit VPN Router
-
```

Comment:
should this profile have the new entity sensor profile in `extends` ?

Reply:
cisco-all-devices has a special case where they combine their proprietary physical entity tables with the generic entity tables.